webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js

```diff
@@ -118,6 +121,7 @@ BreadcrumbBar.defaultProps = {
   loadSwitcherResourcesByResource: noop,
   onSearchChange: noop,
   searchDebounceTimeout: 300,
+  switcherItemOnclick: undefined,
```

Comment:
IMHO, you can remove `switcherItemOnclick` from `defaultProps` as it is not required and anyway by default javascript sets it to `undefined`.